phpstan.neon

```diff
@@ -102,8 +102,15 @@ parameters:
                 - utils/composer-packages/src/Command/AddComposerTypo3ExtensionsToConfigCommand.php
 
         -
-            message: '#Class "Ssch\\TYPO3Rector\\Rector\\__Major__\\__Minor__\\__Name__" is missing @see annotation with test case class reference#'
-            path: utils/generator/templates/src/Rector/__Major__/__Minor__/__Name__.php
+            message: "#^Class \"Ssch\\\\TYPO3Rector\\\\Rector\\\\__Major__\\\\__Minor__\\\\__Name__\" is missing @see annotation with test case class reference$#"
```

Comment:
Could you exclude those files from phpstan analysis. https://phpstan.org/user-guide/ignoring-errors#excluding-whole-files